thunderbird/thunderbird.dockerfile

```diff
@@ -11,9 +11,9 @@ RUN sudo apt-get update \
  && sudo rm -rf /var/lib/apt/lists/*
 
 # Download Thunderbird's source code.
-RUN hg clone --uncompressed https://hg.mozilla.org/comm-central/ thunderbird \
+RUN hg clone --uncompressed https://hg.mozilla.org/mozilla-central/ thunderbird \
  && cd thunderbird \
- && python client.py checkout
+ && hg clone --uncompressed https://hg.mozilla.org/comm-central/ comm \
```

Comment:
Nit: Please don't end a line with `\` if it doesn't continue on the line below.

(This is what caused the CI build to fail.)